rfcs/XXX-eresource_authentication/README.md

```diff
@@ -0,0 +1,59 @@
+# E-Resource Authentication
+
+Wellcome Collection like many other institutions offers its staff and patrons access to electronic journals & books offered by online databases/publishers.
+
+The list of databases offered is [available on the Wellcome Collection site](https://wellcomecollection.org/pages/YDaP2BMAACUAT7DS).
+
+The "Collections and Research" team handles relationships with publishers / database providers, negotiates terms of access and communicate changes in access requirements.
+
+As part of decommissioning the Wellcome Library site we must ensure continuity of access to those resources for patrons and staff.
+
+## E-resources in the Catalogue API
+
+E-resources are currently available from the page linked above, but [can also be found](https://api.wellcomecollection.org/catalogue/v2/works?query=awa6c6gm&include=items) in the catalogue API.
+
+Items will have an `online-resource` location:
+
+![online resource item](online_resource.png)
+
+The URL indexed is for the [EBSCO](https://www.ebsco.com/) Link Resolver
+                           
+EBSCO is a service used by "Collections and Research" to handle database/e-journal metadata.
+
+## Current access methods
+
+### Staff
+
+Access for staff is via [Shibboleth](https://en.wikipedia.org/wiki/Shibboleth_Single_Sign-on_architecture), a profile of [SAML](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language).
+
+We have an on-premises implementation hosted at [idp.wellcome.ac.uk](https://idp.wellcome.ac.uk/idp/shibboleth) and managed by [Overt](https://www.overtsoftware.com/overt-idp/). Staff users authenticate via Active Directory using their Wellcome accounts.
+
+Staff access will be unaffected by the shut down of the Wellcome Library site. 
+
+### Patrons
+
+Library patrons can get access to e-resources by the following methods.
+
+#### Kiosk access
+
+Patrons who are on the library premises can get access from "kiosk" computers that sit on the Wellcome Collection network.
+
+Access to e-resources is achieved by providing publishers / database providers with a fixed IP range from which resources will be accessed. 
+
+The kiosk access method must be maintained and should be unaffected by the shut down of the Wellcome Library site.
+
+The EBSCO link resolver URL that is referenced in Catalogue API results will provide you with a WAM Proxy link that does not require authentication.
```

Comment:
1. What is a WAM Proxy? What is a WAM Proxy link?
2. I'm confused – the way this paragraph was ordered, I thought the fixed IP range was for the library building. If so, why would we go through a proxy? (I think I know why, but it's worth clarifying.)